### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by ATsurkanu**
GasForecasts API and test for it was added.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -42,21 +42,22 @@ task wrapper(type: Wrapper) {
...
-		exclude group: 'org.hibernate', module: 'hibernate-entitymanager'
-	}
-	compile group: 'org.hibernate', name: 'hibernate-core', version: '5.2.10.Final'
-	compile group: 'org.hibernate', name: 'hibernate-entitymanager', version: '5.2.10.Final'
-	compile('javax.mail:javax.mail-api:1.5.6')
-	testCompile('org.springframework.boot:spring-boot-starter-test')
-	testCompile('org.springframework.security:spring-security-test')
-	compile group: 'mysql', name: 'mysql-connector-java', version: '6.0.6'
-	compile group: 'org.json', name: 'json', version: '20160810'
+    compile('org.springframework.boot:spring-boot-starter-web')
```
Comment (parkhomenko) [inline build.gradle]: Did you change anything in this file?

File: src/main/java/com/cityelf/model/Address.java [hunk 2]
```diff
@@ -16,26 +16,25 @@
   @Column(name = "id")
   private long id;
 
-  @Column(name = "street")
-  private String address;
+    @Column(name = "street")
```
Comment (parkhomenko) [inline src/main/java/com/cityelf/model/Address.java]: You don't have to change formatting in existing files.

File: src/main/java/com/cityelf/model/User.java [hunk 3]
```diff
@@ -1,12 +1,11 @@
 package com.cityelf.model;
 
-import java.time.LocalDateTime;
-
 import javax.persistence.Column;
 import javax.persistence.Embedded;
 import javax.persistence.Entity;
 import javax.persistence.Id;
 import javax.persistence.Table;
+import java.time.LocalDateTime;
```
Comment (parkhomenko) [inline src/main/java/com/cityelf/model/User.java]: We should return the order as it was

File: src/main/java/com/cityelf/model/WaterForecast.java [hunk 4]
```diff
@@ -1,104 +1,108 @@
...
-
-  public void setPeopleReport(boolean peopleReport) {
-    this.peopleReport = peopleReport;
-  }
-
-  @Override
-  public boolean equals(Object obj) {
-    if (this == obj) {
-      return true;
+    @Id
```
Comment (parkhomenko) [inline src/main/java/com/cityelf/model/WaterForecast.java]: Again, you changed formatting, no real changes...

**Review by parkhomenko — CHANGES_REQUESTED** [review]
Everything is fine, just lots of issues due to correcting formatting in already existing code.
Also, remove jar libraries from the commit.

File: build.gradle (hunk 1 above)
Comment (ATsurkanu) [reply build.gradle]: No, I did nothing.

File: src/main/java/com/cityelf/model/User.java (hunk 3 above)
Comment (ATsurkanu) [reply src/main/java/com/cityelf/model/User.java]: Ok

File: src/main/java/com/cityelf/model/Address.java (hunk 2 above)
Comment (ATsurkanu) [reply src/main/java/com/cityelf/model/Address.java]: Ok

**Review by parkhomenko — APPROVED** [review]

**Pull request merged**